### PR TITLE
footer delete completed tasks

### PR DESCRIPTION
**Pull request by dbaykina**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -103,6 +108,7 @@ const toggleTask = (target) => {
       }
     }
   });
+  console.log(tasksList);
```
Comment (agolendukhin) [inline index.js]: debug

File: index.js [hunk 2]
```diff
@@ -90,6 +90,11 @@ document.querySelector(".todo-list").addEventListener("click", (e) => {
   }
 });
 
+document.querySelector(".clear-completed").addEventListener('click', () => {
+  //console.log(document.querySelector(".clear-completed"));
```
Comment (agolendukhin) [inline index.js]: it's better not to leave commented code

File: index.js [hunk 3]
```diff
@@ -119,5 +125,18 @@ const countActiveTasks = () => {
   footerCounter.textContent = amount;
 };
 
+const deleteCompletedTasks = () => {
+  
+  tasksList = tasksList.filter((task) => {
```
Comment (agolendukhin) [inline index.js]: `filter(task => !task.completed)`

**Review by agolendukhin — CHANGES_REQUESTED** [review]
A couple of comments to resolve

**Review by nikitamax — APPROVED** [review]

**Pull request merged**